core/src/main/java/com/amplifyframework/api/graphql/error/GraphQLLocation.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.api.graphql.error;
+
+/**
+ * Location mapping to a particular line and column in the request.
+ */
+public class GraphQLLocation {
```

Comment:
So I see this class doesn't have a real constructor that allows instantiation with specific line + column. This is functional for now since we use GSON to directly write these values. Still, we should provide a constructor or static factory method for other plugin implementers that may not rely on a deserializer.

Reply:
Added!